### DISCUSSION TITLE
咨询一下各位： 为什么在使用 ImmortalWrt Firmware Selector 自定 SNAPSHOT 分支的固件的过程中出现 (Error: Missing Checksum) 的错误提示呢？

### DISCUSSION THREAD

**Discussion by dpy013** [immortalwrt/immortalwrt#1274] [Q&A] [answered]
### 问题重现步骤
1.1 首先打开[ImmortalWrt Firmware Selector](https://firmware-selector.immortalwrt.org/)
1.2 在型号 组合框选择 SNAPSHOT 分支
1.3 在型号 可编辑文本输入 Generic x86/64
1.4 选择自定义预安装软件包和首次启动配置脚本 按钮，将需要的软件包加入其中
1.5 基于以上步骤即可重现错误
ps:
使用默认配置编译，也会报(Error: Missing Checksum)错误。
### 下面提供自己整理的配置
zsh curl wget unzip alsa-utils autocore automount base-files block-mount busybox ca-bundle default-settings-chn dnsmasq-full dropbear fdisk firewall4 fstools grub2-bios-setup ipv6helper kmod-8139cp kmod-8139too kmod-ac97 kmod-amd-xgbe kmod-bnx2 kmod-button-hotplug kmod-e1000 kmod-e1000e kmod-forcedeth kmod-fs-f2fs kmod-fs-vfat kmod-i40e kmod-igb kmod-igbvf kmod-igc kmod-ixgbe kmod-ixgbevf kmod-nf-nathelper kmod-nf-nathelper-extra kmod-nft-offload kmod-pcnet32 kmod-r8101 kmod-r8125 kmod-r8168 kmod-sound-hda-codec-hdmi kmod-sound-hda-codec-realtek kmod-sound-hda-codec-via kmod-sound-hda-core kmod-sound-hda-intel kmod-sound-i8x0 kmod-sound-via82xx kmod-tg3 kmod-tulip kmod-usb-audio kmod-usb-hid kmod-usb-net kmod-usb-net-asix kmod-usb-net-asix-ax88179 kmod-usb-net-rtl8150 kmod-usb-net-rtl8152-vendor kmod-vmxnet3 kmod-wireguard libc libgcc libustream-openssl logd luci luci-app-opkg luci-i18n-opkg-zh-cn luci-compat luci-lib-base luci-i18n-base-zh-cn luci-lib-fs luci-lib-ipkg luci-lib-json luci-lib-jsonc luci-proto-ipv6 luci-proto-wireguard luci-theme-material luci-app-firewall luci-i18n-firewall-zh-cn luci-app-ddns-go luci-i18n-ddns-go-zh-cn luci-app-wireguard luci-app-vlmcsd luci-i18n-vlmcsd-zh-cn luci-app-ramfree luci-i18n-ramfree-zh-cn luci-app-diskman luci-i18n-diskman-zh-cn luci-app-alist luci-i18n-alist-zh-cn luci-app-adbyby-plus luci-i18n-adbyby-plus-zh-cn luci-app-passwall luci-i18n-passwall-zh-cn mkf2fs mtd netifd nftables opkg partx-utils ppp ppp-mod-pppoe pppoe-discovery procd procd-seccomp procd-ujail uci uclient-fetch urandom-seed urngd tailscale tcpdump wireguard-tools
### 这是默认的配置
alsa-utils automount base-files busybox ca-bundle dnsmasq-full dropbear fdisk firewall4 fstools grub2-bios-setup ipv6helper kmod-8139cp kmod-8139too kmod-ac97 kmod-amazon-ena kmod-amd-xgbe kmod-bnx2 kmod-button-hotplug kmod-dwmac-intel kmod-e1000 kmod-e1000e kmod-forcedeth kmod-fs-f2fs kmod-fs-vfat kmod-i40e kmod-igb kmod-igbvf kmod-igc kmod-ixgbe kmod-ixgbevf kmod-nft-offload kmod-pcnet32 kmod-r8101 kmod-r8125 kmod-r8168 kmod-sound-hda-codec-hdmi kmod-sound-hda-codec-realtek kmod-sound-hda-codec-via kmod-sound-hda-core kmod-sound-hda-intel kmod-sound-i8x0 kmod-sound-via82xx kmod-tg3 kmod-tulip kmod-usb-audio kmod-usb-hid kmod-usb-net kmod-usb-net-asix kmod-usb-net-asix-ax88179 kmod-usb-net-rtl8150 kmod-usb-net-rtl8152-vendor kmod-vmxnet3 libc libgcc libustream-openssl logd mkf2fs mtd netifd nftables opkg partx-utils ppp ppp-mod-pppoe procd procd-seccomp procd-ujail uci uclient-fetch urandom-seed urngd

**Comment by 1715173329** [comment, marked as answer]
现已修复。

**Comment by dpy013** [comment]
> 现已修复。

感谢，去试试看。
